website/docs/feature-flag-tutorials/use-cases/organize-feature-flags.mdx

```diff
@@ -0,0 +1,349 @@
+Remember that your initial project and environment structure is not set in stone. Unleash is flexible. You can add new environments, create new projects, and even move flags between them later. For larger migrations, you can use the [Import/Export](/how-to/how-to-environment-import-export) functionality or the [Admin API](/api-overview).
+
+The system is designed to evolve with your organization, so don't be afraid to start with a simple setup and refactor as your needs change.
+
+## Design for collaboration and visibility
+
+Modern applications are composed of multiple services with many complex dependencies. While organizing flags into projects is a good starting point, your feature management system should be open by default.
+
+An engineer on the `user-profile-service` team might need to investigate a bug that depends on a flag in the `checkout-api` project.
+When someone searches for a flag, they should be able to view its configuration and ownership so they can request access or submit a change request.
+
+This transparency is key to debugging complex, cross-service interactions. Unleash supports private projects, but these should be the exception, not the norm.
+
+## Secure your setup
+
+Once you have a logical structure for your projects and environments, you can secure it using Unleash's [role-based access control](/reference/rbac) (RBAC) framework. RBAC allows you to implement the principle of least privilege, ensuring users only have the access they need.
+
+Unleash provides two levels of access control:
+- **Root roles**: These roles (Admin, Editor, Viewer, or custom) grant permissions across the entire Unleash instance for global resources like users, groups, SSO configurations, and instance-wide settings.
+- **Project roles**: These roles (Owner, Member, or custom) grant permissions within a specific project. This is where you implement the most granular day-to-day access controls for your development teams.
+
+### Environment-specific permissions
+
+A key capability for enterprise governance is the ability to define custom project roles with environment-specific permissions. This allows you to precisely control what actions a user can take in each environment.
+For example, you can create a "Developer" role that allows a team member to freely create and modify activation strategies in the `development` environment but only allows them to submit a change request for the `production` environment.
+This workflow empowers developers to innovate quickly in safe environments while ensuring that all changes to production are subject to a formal review and approval process.
+
+| Role            | Permissions in development                               | Permissions in test                                       | Permissions in production                                                   | Rationale                                                                                                                                          |
```

Comment:
Is this table an example of possible permissions or how different permissions work?